install/streams.csv

```diff
@@ -1269,7 +1269,7 @@ NBSS,20,200,true,false,true,2011-06-21T00:00:10Z,9999-01-01T00:00:00Z
 NCBS,20,50,true,false,false,2015-06-02T01:45:10Z,9999-01-01T00:00:00Z
 NCBS,20,200,true,false,true,2002-10-07T03:00:00Z,9999-01-01T00:00:00Z
 NCDS,20,50,true,false,false,2013-07-24T01:44:02Z,9999-01-01T00:00:00Z
-NCDS,20,200,true,false,true,2002-04-23T00:00:00Z,9999-01-01T00:00:00Z
+NCDS,20,200,true,false,false,2002-04-23T00:00:00Z,9999-01-01T00:00:00Z
```

Comment:
I'm not sure we have addressed the 200Hz going from triggered to continuous issue yet.  I'd suggest keeping the old stream until we can make a plan for the switch (if we deem it necessary).

Reply:
Thanks @ozym. I placed the 200hz to triggered mode, instead of continuous.